__tests__/app.test.js

```diff
@@ -0,0 +1,148 @@
+    return seed(testData);
+});
+
+afterAll(() => {
+    if (db.end) db.end();
+});
+
+describe('app', () => {
+    describe('GET /...', () => {
+        test('status: 404 when passed an invalid URL', () => {
+            return request(app)
+                .get('/hello')
+                .expect(404);
+        })
+    })
+    describe('GET /api/...', () => {
+        test('status: 404 when passed an invalid path', () => {
+            return request(app)
+                .get('/api/animals')
+                .expect(404);
+        })
+    })
+    describe('GET /api/topics', () => {
+        test('status: 200', () => {
+            return request(app)
+                .get('/api/topics')
+                .expect(200);
+        })
+        test('status: 200 and responds with an object', () => {
+            return request(app)
+            .get('/api/topics')
+            .expect(200)
+            .then((response) => {
+                const result = response.body;
+                expect(typeof result).toBe('object');
+            })
+        })
+        test('status: 200 and responds with an object with a key of topics', () => {
+            return request(app)
+            .get('/api/topics')
+            .expect(200)
+            .then((response) => {
+                const result = response.body;
+                expect(result).toHaveProperty('topics');
+            }) 
+        })
+        test('status: 200 and responds with a nested array containing all the topic objects', () => {
+            return request(app)
+            .get('/api/topics')
+            .expect(200)
+            .then((response) => {
+                const result = response.body.topics;
+                expect(Array.isArray(result)).toBe(true);                
+                expect(result).toHaveLength(3);
+             })
+        })
+        test('status: 200 and responds with a nested array containing all the topic objects with the correct keys', () => {
+            return request(app)
+            .get('/api/topics')
+            .expect(200)
+            .then((response) => {
+                const result = response.body.topics;
+                result.forEach((topic) => {
+                    expect(topic).toHaveProperty("description");
+                    expect(topic).toHaveProperty("slug");
+                })
+             })
+        })
+    })
+    describe('GET /api/articles', () => {
+        test('status: 200', () => {
+            return request(app)
+                .get('/api/articles')
+                .expect(200);
+        })
+        test('status: 200 and responds with an object', () => {
+            return request(app)
+            .get('/api/articles')
+            .expect(200)
+            .then((response) => {
+                const result = response.body;
+                expect(typeof result).toBe('object');
+            })
+        })
+        test('status: 200 and responds with an object with a key of topics', () => {
```

Comment:
Just make sure you're test descriptions are right this is the articles endpoint now, not topics